.travis.yml

```diff
@@ -30,12 +30,14 @@ matrix:
     - php: 7.2
     - php: 7.3
     - php: 7.4
+    - php: 8.0
+      dist: bionic
     - php: nightly
       env:
         - COMPOSER_DEV_STABILITY=true
 
 before_install:
-  - sudo add-apt-repository ppa:mc3man/trusty-media -y
+  - sudo add-apt-repository ppa:mc3man/${TRAVIS_DIST}-media -y
```

Comment:
@kornrunner Quick question here, will ${TRAVIS_DIST} properly match the `dist` parameter set up before?
This is for my information, I've never seen this type of variable before.

Reply:
Yes, that's picked up from https://docs.travis-ci.com/user/environment-variables/. You can see the replaced value in logs as well: https://travis-ci.org/github/PHP-FFMpeg/PHP-FFMpeg/jobs/748664569